themes/ddbasic/sass/components/webform.scss

```diff
@@ -0,0 +1,51 @@
+@import '../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?